test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
```

Comment:
use rspec for tests.
remove test folder.
All test should be stored in /spec folder.